config/config.go

```diff
@@ -56,33 +55,43 @@ func Load(file string) (config ProjectConfig, err error) {
 		return config, err
 	}
 	err = yaml.Unmarshal(data, &config)
-	config = fix(config)
-	config, err = fillGitData(config)
-	if err != nil {
+	config.fillBasicData()
+	if err := config.fillFiles(); err != nil {
+		return config, err
+	}
+	if err := config.fillGitData(); err != nil {
 		return config, err
 	}
+	return config, config.validade()
+}
+
+func (config *ProjectConfig) validade() (err error) {
 	if config.BinaryName == "" {
-		return config, errors.New("missing binary_name")
+		return errors.New("missing binary_name")
```

Comment:
You could, if you wanted, replace these `errors.New` statements with `fmt.Errorf`.

But that seems way out of scope for this PR